.gitmodules

```diff
@@ -19,6 +19,9 @@
 [submodule "extern/adpcm"]
 	path = extern/adpcm
 	url = https://github.com/superctr/adpcm
+[submodule "extern/cam900_vgsound_emu"]
+	path = extern/cam900_vgsound_emu
+	url = https://github.com/cam900/vgsound_emu
```

Comment:
I suggest you to not use submodules (especially if it's just two/three files), or at least specify `branch = main`. My default branch still is master.

Reply:
That's for future expansion, it will be added additional chips, ex: N163, ES5506, etc...

Reply:
...or I can borrow emulation from there when i adding new systems on furnace.